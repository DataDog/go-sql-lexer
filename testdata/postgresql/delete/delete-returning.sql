DELETE FROM orders WHERE id = 8 RETURNING *;
