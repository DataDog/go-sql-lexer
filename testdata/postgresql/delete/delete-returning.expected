[
    {
        "expected": "DELETE FROM orders WHERE id = ? RETURNING *",
        "statement_metadata": {
            "size": 12,
            "tables": [
                "orders"
            ],
            "commands": [
                "DELETE"
            ],
            "comments": [],
            "procedures": []
        }
    }
]